### ISSUE TITLE
Update multimatch

### ISSUE THREAD

**Issue by mahnunchik** [segmentio/metalsmith-ignore#9]
Could you please update `multimatch` module to remove warning:
```
npm WARN deprecated minimatch@0.2.14: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue

```

**Comment by woodyrew** [comment]
Addressed with PR #11

**Comment by woodyrew** [comment]
This can be closed when #11 is merged. Is there a timescale when this might happen?
